openstudiocore/src/model/CoilWaterHeatingDesuperheater_Impl.hpp

```diff
@@ -40,6 +40,7 @@ class Schedule;
 class CurveBiquadratic;
 class HVACComponent;
 class WaterHeaterMixed;
+class WaterHeaterStratified;
```

Comment:
I understand why you added `WaterHeaterStratified` here, but the `WaterHeaterMixed` forward declaration wasn't needed to begin with, so I will remove these two.